src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private final int figureCount = 50;

```

Comment:
if is it class constant, it should be private static final and names in UPPER_CASE